app/controllers/this_is_cool_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ThisIsCoolController < ApplicationController
+    before_action :authenticate_user!
+
+    def create
+        ThisIsCool.create!( project_id: params[:project_id], user: current_user)
+        redirect_to :back
+    end
+
```

Comment:
I think it looks a lot nicer when there is no empty space left at the end of the file/class